roles/neutron-common/templates/etc/neutron/policy.json

```diff
@@ -36,8 +46,11 @@
     "get_network:provider:physical_network": "rule:admin_only",
     "get_network:provider:segmentation_id": "rule:admin_only",
     "get_network:queue_id": "rule:admin_only",
+    "get_network_ip_availabilities": "rule:admin_only",
+    "get_network_ip_availability": "rule:admin_only",
     "create_network:shared": "rule:admin_only",
     "create_network:router:external": "rule:admin_only",
+    "create_network:is_default": "rule:admin_only",
```

Comment:
This is part of the "give me a network" spec, and we may consider allowing cloud_admin to do this. http://docs.openstack.org/mitaka/networking-guide/intro-os-networking-features.html

This allows creating network/subnet/router on demand when a tenant (first) boots a VM.

This could be delayed to 3.1 as well.